README.md

```diff
@@ -277,14 +277,9 @@ In order to run full test suite, simply execute:
 **NOTE**: Requires `tox` (please install with `pip install tox`), several supported versions of Python on your path, and `librdkafka` [installed](tools/bootstrap-librdkafka.sh) into `tmp-build`.
 
 
-**Run integration tests:**
-
-To run the integration tests, uncomment the following line from `tox.ini` and add the paths to your Kafka and Confluent Schema Registry instances. You can also run the integration tests outside of `tox` by running this command from the source root.
-
-    examples/integration_test.py <kafka-broker> [<test-topic>] [<schema-registry>]
-
-**WARNING**: These tests require an active Kafka cluster and will create new topics.
+**Integration tests:**
 
+    See [tests/README.md](tests/README.md) for instructions on how to run integration tests.
```

Comment:
Look at the rendered file and you'll see that 4 space indent gives you monospace, which is not intended here. 0 indent! :)